platform/el/build-image.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Should have probably been just a Dockerfile ...
```

Comment:
Given this comment, what would you change? We're already providing containerised versions of the server/client tooling. 

Do you mean a containerised build environment for the EL-specific packages? That'd  be preferred either way...